packages/medusa/src/api/routes/admin/discounts/update-discount.ts

```diff
@@ -3,6 +3,7 @@ import {
   IsArray,
```

Comment:
comment: Looping in @srindom to make sure we are aligned on this decision; disallowing `type` + `is_dynamic` updates to discounts

Reply:
Think it seems reasonable! 👍